src/quartz_mailer/message.cr

```diff
@@ -0,0 +1,98 @@
+require "email"
+
+class Quartz::Message
+  alias Address = EMail::Address
+
+  getter _from : Address?
+  getter _to   = [] of Address
+  getter _cc   = [] of Address
+  getter _bcc  = [] of Address
+
+  getter _subject = ""
+  getter _text = ""
+  getter _html = ""
+
+  def self.address(email : String)
+    Address.new email
+  end
+
+  def self.address(name : String, email : String)
+    Address.new email, name
+  end
+
+  def from(email : String) : Nil
+    from Message.address email
+  end
+
+  def from(name : String, email : String) : Nil
+    from Message.address name, email
+  end
+
+  def from(@_from : Address)
+  end
+
+  {% for destination_field in [:to, :cc, :bcc] %}
+    {% destination_field = destination_field.id %}
+
+    def {{ destination_field }}(email : String) : Nil
+      @_{{ destination_field }} << Message.address email
+    end
+
+    def {{ destination_field }}(name : String, email : String) : Nil
+      @_{{ destination_field }} << Message.address name, email
+    end
+
+    def {{ destination_field }}(address : Address) : Nil
+      @_{{ destination_field }} << address
+    end
+
+    def {{ destination_field }}(addresses : Array(Address)) : Nil
+      addresses.each do |address|
+        {{ destination_field }} address
+      end
+    end
+  {% end %}
+
+  def subject(@_subject : String) : Nil
+  end
+
+  def text(@_text : String) : Nil
+  end
+
+  def html(@_html : String) : Nil
+  end
+
+  # for backwards compatibility
+  def body(@_text : String) : Nil
+  end
+
+  def to_email
+    EMail::Message.new.tap do |email|
```

Comment:
I was hoping to find an answer here: https://crystal-lang.org/docs/conventions/coding_style.html

But it doesn't seem to cover this case. It may be worth asking in the crystal chat room if there is an agreed upon style.

Reply:
The non-scientific consensus in crystal chat seems to be `Email`

Reply:
I agree that it should have a lowercase "m". In this case the class being wrapped, from the crystal-email shard, is [defined as EMail](https://github.com/arcage/crystal-email/blob/master/src/email.cr#L10) so I thought I should be consistent.

The Message class here completely wraps the EMail::Message class, so an Amber user shouldn't ever need to deal with it.